### PR TITLE
Added options to run as a CLI tool

### PR DESCRIPTION
**Pull request by ChrisCates**
This is just a handy hotfix to make it accessible as CLI tool.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -4443,7 +4456,7 @@
       "requires": {
         "async": "^2.1.5",
         "find-process": "^1.0.5",
-        "fs-ext": "github:baudehlo/node-fs-ext#7c9824f3dc330e795aa13359d96252860bd3a684",
+        "fs-ext": "github:baudehlo/node-fs-ext#2ba366d9fc67ef3ab165e239068924b276ecf249",
```
Comment (oed) [inline package-lock.json]: Why is this needed?

Comment (ChrisCates) [reply package-lock.json]: Hmm... Seems completely redundant. Must have been updated when I ran npm install.

Ill remove it from the commit.

Comment (oed) [reply package-lock.json]: oh, wait I thought it was in `package.json` but it's in the package lock. NVM then!
